### PR TITLE
DEV-672 - Concept page for delegated repositories

### PR DESCRIPTION
**Pull request by ZachGoldberg**

### CODE REVIEW

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+:::note
+Delegated Repositories are only available to DevOps Foundations Enterprise customers.
+:::
+
+As enterprises scale their usage of IaC across more teams there often arises a need to enforce least-privilege access control to both IaC source code and the CI/CD of that code. Gruntwork's recommendation is to leverage multiple source code repositories to create hard-boundaries that enforce least-privilege access control. The pattern we recommend consists of three parts:
+
+1. A single `infrastructure-live-root` repository, governed by the core platform team. This repository includes IaC for security and shared infrastructure including core-account creation, central logging/auditing and networking controls (e.g. GuardDuty, SecurityHub, Macie, Transit Gateway etc.).
+1. Many _delegated_ repositories, often named using the pattern `infrastructure-live-$TEAM_NAME`, where individual teams can manage their own IaC. These repositories are created by the core platform team using the Account Factory and, via `infrastructure-live-access-control` are granted permissions to manage their own infrastructure.
+git
```
Comment (Resonance1584) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:11]:
```suggestion

```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Using this pattern central platform teams have the ability to:
+* Centrally manage core compliance and security infrastructure
+* Centrally manage which teams have permission to deploy what kind of infrastructure, allowing central teams to enforce compliance with approved architecture-board approved plans
```
Comment (Resonance1584) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:14]:
```suggestion
* Centrally manage which teams have permission to deploy what kind of infrastructure, allowing their team to enforce compliance with architecture-board approved plans
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* Centrally manage tags for shared and per-team infrastructure
+* Enforce least-privilege access to IaC, restricting teams to only have access only to IaC and deployment capability / role assumptions for resources that they are responsible for.
```
Comment (Resonance1584) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:16]:
```suggestion
* Enforce least-privilege access to IaC, restricting teams to have access only to IaC and deployment capability / role assumptions for resources that they are responsible for.
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Delegated Repository Creation
+Delegated repositories are optionally created by [Account Factory](http://localhost:3000/2.0/docs/accountfactory/concepts/) during account creation.  A delegated account vend follows the following (automated) workflow:
```
Comment (Resonance1584) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:19]:
```suggestion
Delegated repositories are optionally created by [Account Factory](/2.0/docs/accountfactory/concepts/) during account creation.  A delegated account vend follows the following (automated) workflow:
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Delegated Repositoires
+
+:::note
+Delegated Repositories are only available to DevOps Foundations Enterprise customers.
+:::
+
+As enterprises scale their usage of IaC across more teams there often arises a need to enforce least-privilege access control to both IaC source code and the CI/CD of that code. Gruntwork's recommendation is to leverage multiple source code repositories to create hard-boundaries that enforce least-privilege access control. The pattern we recommend consists of three parts:
+
+1. A single `infrastructure-live-root` repository, governed by the core platform team. This repository includes IaC for security and shared infrastructure including core-account creation, central logging/auditing and networking controls (e.g. GuardDuty, SecurityHub, Macie, Transit Gateway etc.).
+1. Many _delegated_ repositories, often named using the pattern `infrastructure-live-$TEAM_NAME`, where individual teams can manage their own IaC. These repositories are created by the core platform team using the Account Factory and, via `infrastructure-live-access-control` are granted permissions to manage their own infrastructure.
```
Comment (oredavids) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:10]:
```suggestion
1. Many _delegated_ repositories, often named using the pattern `infrastructure-live-$TEAM_NAME`, where individual teams can manage their own IaC. These repositories are created using the Account Factory and are granted permissions to manage their own infrastructure via `infrastructure-live-access-control`.
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md (hunk 1 above)
Comment (oredavids) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:11]:
```suggestion

```
Typo?

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+git
+Using this pattern central platform teams have the ability to:
```
Comment (oredavids) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:12]:
```suggestion
Using this pattern core platform teams have the ability to:
```
For consistency

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# Delegated Repositoires
+
+:::note
+Delegated Repositories are only available to DevOps Foundations Enterprise customers.
```
Comment (yhakbar) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:4]: Our wording here is really tricky. 

You can see [here](https://github.com/gruntwork-io/docs/blob/main/docs/2.0/docs/pipelines/guides/setup-delegated-repo.mdx) that I tried to call this "Account Factory vended delegated repos" but maybe we need a new term for what that that other thing is. Users can have some amount of infrastructure delegation without being enterprise customers, they just don't get Account Factory vending them as part of account creation.

Whatever we call the other thing, it's an important thing for users to know about, as many want to know how to handle things like application deployments that require AWS access to push artifacts, etc, even if they aren't managing IaC.

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md (hunk 2 above)
Comment (oredavids) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:14]:
```suggestion
* Centrally manage which teams have permission to deploy different types of infrastructure, allowing core teams to enforce compliance with approved architecture-board approved plans
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md (hunk 4 above)
Comment (oredavids) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:19]:
```suggestion
Delegated repositories are optionally created by [Account Factory](/2.0/docs/accountfactory/concepts) during account creation.  A delegated account vend follows the following (automated) workflow:
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md (hunk 5 above)
Comment (yhakbar) [inline docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:10]:
```suggestion
1. Many _delegated_ repositories, often named using the pattern `infrastructure-live-$TEAM_NAME`, where individual teams can manage their own IaC. These repositories are created by the core platform team using the Account Factory and, via `infrastructure-live-access-control` are granted limited permissions to manage their own infrastructure.
```

File: docs/2.0/docs/accountfactory/concepts/delegated-repositories.md (hunk 2 above)
Comment (yhakbar) [reply docs/2.0/docs/accountfactory/concepts/delegated-repositories.md:14]: I would recommend we just explicitly say "allowing the central platform team to enforce". Using different terms for the team doing the regulation gets hard to follow.

**Review by oredavids — APPROVED** [review]

**Review by yhakbar — APPROVED** [review]
